pom.xml

```diff
@@ -68,11 +73,30 @@
 			<artifactId>core-utils-lib</artifactId>
 			<version>1.7.0</version>
 		</dependency>
-        <dependency>
-            <groupId>org.spockframework</groupId>
-            <artifactId>spock-core</artifactId>
-            <scope>test</scope>
-        </dependency>
+		<dependency>
+				<groupId>org.spockframework</groupId>
+				<artifactId>spock-core</artifactId>
+				<version>2.0-M4-groovy-3.0</version>
+				<scope>test</scope>
+		</dependency>
+
+		<!-- use groovy 3 -->
+		<dependency>
+			<groupId>org.codehaus.groovy</groupId>
+			<artifactId>groovy-all</artifactId>
+			<version>3.0.7</version>
+			<type>pom</type>
+		</dependency>
+		<dependency>
+			<groupId>org.codehaus.groovy</groupId>
+			<artifactId>groovy-sql</artifactId>
+			<version>3.0.7</version>
+		</dependency>
+		<dependency>
+			<groupId>org.codehaus.groovy</groupId>
+			<artifactId>groovy-json</artifactId>
+			<version>3.0.7</version>
+		</dependency>
```

Comment:
I'm pretty sure that this dependency is unnecessary

Reply:
I am not so sure. The `parent-pom` specifies this version. If we do not overwrite it here, we still end up with `groovy-json:2.5.7` in the mix

Reply:
Ah right, but then it should be removed once the parent-pom is removed from the sample-tracking-update-portlet